usage-collection/src/main/proto/usage.proto

```diff
@@ -3307,10 +3357,10 @@ message CDPStackPatchEvent {
 // Type of stack patch event.
 message CDPStackPatchEventType {
   enum Value {
-    UNSET = 0;
-    AFFECTED = 1;
-    SUCCESS = 2;
-    FAILURE = 3;
+      UNSET = 0;
+      AFFECTED = 1;
+      SUCCESS = 2;
+      FAILURE = 3;
   }
```

Comment:
Like above, this indentation looks incorrect.

Reply:
this is copy paste from thunderhead